src/renderer/components/Modal/Profile.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <modal name="profile-modal" @before-open="beforeOpen">
+        <h1>Create a profile</h1>
+
+        <p>
+            <label for="column-name"> Profile Name</label>
+            <input type="text" for="column-name" v-model="profile_name"/>
+        </p>
+
+        <p>
+            <button @click="createColumn">Create profile</button>
+        </p>
+    </modal>
+</template>
+
+<script>
+    export default {
+        name: 'profile-modal',
+        data: function () {
+            return {
+                profile_name: ''
```

Comment:
While it is okay style-wise, I overall used `camelCase` for values, properties and method names.

So please change this to `profileName` instead!